src/Mod/Sketcher/Gui/TaskSketcherElements.cpp

```diff
@@ -416,27 +591,12 @@ void TaskSketcherElements::onSelectionChanged(const Gui::SelectionChanges& msg)
                     }
                 }
                 // update the listwidget
-                int element=ui->comboBoxElementFilter->currentIndex();
                 ui->listWidgetElements->blockSignals(true);

```

Comment:
This should be fixed even if not part of the PR

Reply:
You mean turning : 

    ui->listWidgetElements->blockSignals(true);
    ui->listWidgetElements->clearSelection ();
    ui->listWidgetElements->blockSignals(false);

into 

    bool sgnVal = ui->listWidgetElements->blockSignals(true);
    ui->listWidgetElements->clearSelection ();
    ui->listWidgetElements->blockSignals(sgnVal );

? There are a lot of those. Should I change all?

Reply:
Hmmm. Yes there are lot. Maybe another commit or PR is needed. :?

Reply:
There's not so many, about 4-5. I don't think it's worth making separate pr as most of the file is changed here.